packages/replay/src/util/getPrivacyOptions.ts

```diff
@@ -79,7 +79,12 @@ export function getPrivacyOptions({
     maskInputSelector: maskSelector,
     unmaskInputSelector: unmaskSelector,
 
-    blockSelector: getOption(block, ['.sentry-block', '[data-sentry-block]'], blockClass, blockSelector),
+    blockSelector: getOption(
+      block,
+      ['.sentry-block', '[data-sentry-block]', 'base[href="/"]'],
```

Comment:
While this makes sense, it feels a bit "weird" and maybe confusing to put these things together like this - as the first two are really custom classes/attributes that a user can set to manually unblock things, while the latter is something we just want to generally block.

I would propose to extract the new selector into a variable, which makes this a bit clearer:

```js
const defaultBlockedElements = ['base[href="/"]'];
// ...
getOption(
  block, 
  ['.sentry-block', '[data-sentry-block]', ...defaultBlockedElements], 
  blockClass, 
  blockSelector
);
```

We may add more here later, and then it is IMHO clearer what's going on.